### ISSUE TITLE
Default use umd bundle in package.json

### ISSUE THREAD

**Issue by tamascsaba** [scttcper/ngx-toastr#66]
In package.json main filed toastr.js is not commonjs compatible, pls use umd or commonjs bundle in main filed and use module field for ES6 build like [angular](https://github.com/angular/angular/blob/master/modules/%40angular/core/package.json)

```
{
  "main": "./toastr.umd.js",
  "module": "./toastr.js",
  "jsnext:main": "./toastr.js",
  "typings": "./toastr.d.ts",
}
```

**Comment by tamascsaba** [comment]
thx for quick solution :+1:

**Comment by scttcper** [comment]
Thanks for doing the legwork on that. released in 4.2.1
